build/secondary/third_party/dart/pkg/meta/BUILD.gn

```diff
@@ -0,0 +1,25 @@
+# This file is generated by package_importer.py for meta-1.4.0
+
+# TODO(flutter/flutter#85356): `package_importer.py` is part of the fuchsia.git
```

Comment:
I don't think generating and checking in the BUILD.gn files from manifests is the right way to automating the inference of build rules for GN. It is entirely possible stamp rules at GN gen time and reference them in other targets. Flutter does (or used to do this) for the Fuchsia SDK for instance. Instead of generating and checking in BUILD.gn files for each target, targets were generated during gen time from the JSON manifests. Food for thought.

Reply:
Agreed. I updated the TODO comment, and the issue, to recommend generating the targets in general, without prescribing that we generate the BUILD.gn files themselves.